src/io/flutter/view/FlutterPerfView.java

```diff
@@ -207,7 +206,7 @@ private void addPerformanceViewContent(FlutterApp app, ToolWindow toolWindow) {
 
       addPerformanceTab(runnerTabs, app, toolWindow, true);
 
-      if (FlutterUtils.isAndroidStudio() && !FlutterSettings.getInstance().isMemoryProfilerDisabled()) {
```

Comment:
Should we silently remove the memory tab? Or would you like to wait until we either launch DevTool automatically or a link on how to use DevTools for profiling?

Reply:
We should improve the way we launch devtools slightly. Perhaps open an issue to remove the memory view for the next release?